README

```diff
@@ -1,4 +1,13 @@
-gxds is a set of Go packages which can be used to build
-a Go-based EdgeX Foundry device service.
+# Go Device Service SDK
+## Overview
+This repository is a set of Go packages which can be used to build a Go-based EdgeX Foundry Device Service.
+## Usage
+Developers could make their own Device Service by implementing the `ProtocolDriver  interface` for the specific IoT protocol and main function to start the Device Service.  To implement the main function, the startup package could be optional leveraged, or developers could write the customized bootstrap code by themselves.
+Please see the build-in [Simple Device Service](https://github.com/edgexfoundry/device-sdk-go/tree/master/example) as an example.
 
-This is a work-in-progress...
+## Community
+- Chat: https://chat.edgexfoundry.org/home
+- Mainling lists: https://lists.edgexfoundry.org/mailman/listinfo
```

Comment:
Mainling → Mailing

Reply:
@jduranf 
Thanks for the review!
In fact, I copied this section from edgex-go, so the same type there...
https://github.com/edgexfoundry/edgex-go/blob/master/README.md

Reply:
https://github.com/edgexfoundry/edgex-go/issues/849 opened for edgex-go

Reply:
@jduranf fixed the typo